unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -0,0 +1,308 @@
+            Clear();
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestOwnedWearablesAsync(string userId, int pageNumber, int pageSize, bool cleanCachedPages, CancellationToken ct)
+        {
+            var createNewPointer = false;
+            if (!ownerWearablesPagePointers.TryGetValue(userId, out var pagePointer))
+            {
+                createNewPointer = true;
+            }
+            else if (cleanCachedPages)
+            {
+                pagePointer.Dispose();
+                ownerWearablesPagePointers.Remove(userId);
+                createNewPointer = true;
+            }
+
+            if (createNewPointer)
+            {
+                ownerWearablesPagePointers[userId] = pagePointer = new (
+                    WEARABLES_BY_OWNER_END_POINT.Replace("{userId}", userId),
+                    pageSize, ct, this);
+            }
+
+            var pageResponse = await pagePointer.GetPageAsync(pageNumber, ct);
+            AddWearablesToCatalog(pageResponse.response.wearables);
+
+            return pageResponse.response.wearables;
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestBaseWearablesAsync(CancellationToken ct)
+        {
+            var serviceResponse = await lambdasService.Ref.Get<WearableResponse>(
+                "",
+                WEARABLES_BY_COLLECTION_END_POINT.Replace("{collectionId}", BASE_WEARABLES_COLLECTION_ID),
+                REQUESTS_TIME_OUT_SECONDS,
+                ATTEMPTS_NUMBER,
+                ct,
+                LambdaPaginatedResponseHelper.GetPageSizeParam(1),
+                LambdaPaginatedResponseHelper.GetPageNumParam(1));
+
+            AddWearablesToCatalog(serviceResponse.response.wearables);
+
+            return serviceResponse.response.wearables;
+        }
+
+        public async UniTask<IReadOnlyList<WearableItem>> RequestThirdPartyWearablesByCollectionAsync(string userId, string collectionId, int pageNumber, int pageSize, bool cleanCachedPages, CancellationToken ct)
+        {
+            var createNewPointer = false;
+            if (!thirdPartyCollectionPagePointers.TryGetValue((userId, collectionId), out var pagePointer))
+            {
+                createNewPointer = true;
+            }
+            else if (cleanCachedPages)
+            {
+                pagePointer.Dispose();
+                thirdPartyCollectionPagePointers.Remove((userId, collectionId));
+                createNewPointer = true;
+            }
+
+            if (createNewPointer)
+            {
+                thirdPartyCollectionPagePointers[(userId, collectionId)] = pagePointer = new (
+                    WEARABLES_BY_THIRD_PARTY_COLLECTION_END_POINT
+                       .Replace("{userId}", userId)
+                       .Replace("{collectionId}", collectionId),
+                    pageSize, ct, this);
+            }
+
+            var pageResponse = await pagePointer.GetPageAsync(pageNumber, ct);
```

Comment:
You should handle `pageResponse.success`

Reply:
Done: https://github.com/decentraland/unity-renderer/pull/4084/commits/8b240ddbd614d8149c720b4b3b2e02086ba0d7ca